api/src/sendinblue.js

```diff
@@ -28,9 +28,10 @@ const api = async (path, options = {}) => {
 async function sendEmail(to, subject, htmlContent, { params, attachment, cc, bcc } = {}) {
   try {
     const body = {};
-
+    console.log('DATA', to, cc);
     body.to = [to];
     if (cc?.length) body.cc = cc;
+    body.cc = [cc];
```

Comment:
I guess it's risky, we have introduced a breaking change here. In some other places of code, users may have sent an array of `cc` (see line 30 which consider `cc` is an array). For instance, this code would break 😱 :

https://github.com/betagouv/service-national-universel/blob/e697e5d774bb26aa37fefc12d453ace7d0ca94e0/api/src/utils/index.js#L130-L150

I would suggest removing this line.

Reply:
Yes, and as your suggestion provides the cc as an array earlier in the code, there's no need to change something here as it could break something indeed